recipes/approvaltests.cpp/config.yml

```diff
@@ -0,0 +1,5 @@
+versions:
+  8.0.0:
+    folder: all
+  8.1.0:
+    folder: all
```

Comment:
Is it right that there are two identical config.yml files here?
* recipes/approvaltests.cpp/config.yml
* recipes/approvaltests.cpp/all/config.yml

Reply:
That's a goof. Thank you. Nice catch!